apps/server/src/employees/employees.service.ts

```diff
@@ -22,7 +22,7 @@ export class EmployeesService {
         positionId: createEmployeeDto.positionId,
         pswdHash: await bcrypt.hash(
           createEmployeeDto.password,
-          process.env.SALT_ROUNDS || 10,
+          await bcrypt.genSalt(10),
```

Comment:
can we use the env variable with `10` as the fallback?

Reply:
For that, we will need to use ConfigService and that will add a dependency to the folder. Should I try that?

Reply:
yep, that makes sense